src/BlazorBoilerplate.Server/Startup.cs

```diff
@@ -80,10 +91,18 @@ public void ConfigureServices(IServiceCollection services)
               .AddConfigurationStore(options =>
               {
                   options.ConfigureDbContext = builder => {
-                      if (Convert.ToBoolean(Configuration["BlazorBoilerplate:UseSqlServer"] ?? "false"))
+                      if (!String.IsNullOrWhiteSpace(Configuration.GetValue<string>("DOCKER_COMPOSE_SQL")))
+                      {
+
```

Comment:
Can you please remove these empty lines. I like to keep the code clean and readable.

Reply:
Done